sektionen/sammansattning/body.md

```diff
@@ -9,12 +7,12 @@ kontaktfunktionär.
 
 En förteckning över funktionärernas kontaktperson i styrelsen finns [här](https://dsekt.se/kontaktperson).
 
-## Ordförande - Adam Sjöberg
+## Sektionsordförande - Adam Sjöberg
```

Comment:
These changes would be nice to keep, but probably best to leave that to the original Authors to fix

Reply:
Whoops